### PR TITLE
[BYOC][ACL] removed ACL 20.05 limitations

### PR DESCRIPTION
**Pull request by d-smirnov**
This PR removes checks for padding in according with changes in ACL 20.11 which currently does not require padded data.

### CODE REVIEW

**Comment by d-smirnov** [comment]
@mbaret

File: tests/python/contrib/test_arm_compute_lib/test_dense.py [hunk 1]
```diff
@@ -101,7 +101,7 @@ def _get_qnn_model(
     out = relay.qnn.op.requantize(
         out,
         relay.const(input_sc * kernel_sc, "float32"),  # input scale
-        relay.const(input_zp * kernel_zp, "int32"),  # input zero point
+        relay.const(0, "int32"),  # input zero point
```
Comment (mbaret) [inline tests/python/contrib/test_arm_compute_lib/test_dense.py:104]: What's the relevance of this change?

**Review by mbaret — COMMENTED** [review]
This looks OK, but a question on what's happened with the tests. It seems like you've removed the check for the number of ACL partitions - is this because previously some tests wouldn't offload due to the padding restriction but now they always do?

File: tests/python/contrib/test_arm_compute_lib/test_dense.py (hunk 1 above)
Comment (d-smirnov) [reply tests/python/contrib/test_arm_compute_lib/test_dense.py:104]: Should be zero, as dense already counted all the input zero points in.

**Comment by d-smirnov** [comment]
Correct. Tests were restored to initial state (before padding limitations). Now each test always resulted to ACL-partitioned IR so there is no need to have this variations.

**Comment by d-smirnov** [comment]
Ping. How we can make a progress here?

**Review by mbaret — APPROVED** [review]
This LGTM.

**Comment by mbaret** [comment]
Thanks @d-smirnov

**Pull request merged**